### PR TITLE
doc: fix typo

### PR DESCRIPTION
**Pull request by xiexingguo**
Signed-off-by: xie xingguo <xie.xingguo@zte.com.cn>

### CODE REVIEW

**Comment by xiexingguo** [comment]
@tchaikov Some nits. Do you mind having a look at? Thanks

**Comment by xiexingguo** [comment]
@tchaikov Ping

**Comment by xiexingguo** [comment]
@tchaikov Ping. Or you can just close it out if it turns out to be irrelevant.    Thanks a lot:-)

File: doc/dev/index.rst [hunk 1]
```diff
@@ -34,7 +34,7 @@ notice that a link is broken or if you know of a better link, please
 
 .. _`report it as a bug`: http://tracker.ceph.com/projects/ceph/issues/new
 
-Essentials (tl;dr)
+Essentials
```
Comment (tchaikov) [inline doc/dev/index.rst]: why remove this? "tl;dr" is short for "too long; don't read".

**Comment by tchaikov** [comment]
@xiexingguo sorry, thought it's low priority so failed to look at it earlier.

File: doc/rados/api/librados-intro.rst [hunk 2]
```diff
@@ -353,7 +353,6 @@ you to initialize a ``librados::Rados`` cluster handle object:
 			ret = cluster.init2(user_name, cluster_name, flags);
 			if (ret < 0) {
 				std::cerr << "Couldn't initialize the cluster handle! error " << ret << std::endl;
-				ret = EXIT_FAILURE;
 				return 1;
```
Comment (tchaikov) [inline doc/rados/api/librados-intro.rst]: better off `return EXIT_FAILURE;`

File: doc/rados/api/librados-intro.rst [hunk 3]
```diff
@@ -365,7 +364,6 @@ you to initialize a ``librados::Rados`` cluster handle object:
 			ret = cluster.conf_read_file("/etc/ceph/ceph.conf");	
 			if (ret < 0) {
 				std::cerr << "Couldn't read the Ceph configuration file! error " << ret << std::endl;
-				ret = EXIT_FAILURE;
 				return 1;
```
Comment (tchaikov) [inline doc/rados/api/librados-intro.rst]: ditto.

File: doc/rados/api/librados-intro.rst [hunk 4]
```diff
@@ -377,7 +375,6 @@ you to initialize a ``librados::Rados`` cluster handle object:
 			ret = cluster.conf_parse_argv(argc, argv);
 			if (ret < 0) {
 				std::cerr << "Couldn't parse command line options! error " << ret << std::endl;
-				ret = EXIT_FAILURE;
 				return 1;
```
Comment (tchaikov) [inline doc/rados/api/librados-intro.rst]: ditto.

File: doc/rados/api/librados-intro.rst [hunk 5]
```diff
@@ -389,7 +386,6 @@ you to initialize a ``librados::Rados`` cluster handle object:
 			ret = cluster.connect();
 			if (ret < 0) {
 				std::cerr << "Couldn't connect to cluster! error " << ret << std::endl;
-				ret = EXIT_FAILURE;
 				return 1;
```
Comment (tchaikov) [inline doc/rados/api/librados-intro.rst]: the same.

File: doc/dev/index.rst (hunk 1 above)
Comment (xiexingguo) [inline doc/dev/index.rst]: Aha, I thought they are just some gibberish. Thank you for your explanation, will drop it.

**Comment by xiexingguo** [comment]
All fixed. Thanks  for the review.

**Comment by tchaikov** [comment]
@xiexingguo would be ideal to be more specific in the commit message and its title.

**Comment by xiexingguo** [comment]
Thank you for your reminder. Next time I'll try to make myself crystal clear and don't be lazy:)

**Pull request merged**